samples/matter/lock/src/credentials/credentials_data_types.cpp

```diff
@@ -176,4 +176,150 @@ CHIP_ERROR User::Deserialize(const void *buff, size_t buffSize)
 	return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR WeekDaySchedule::FillFromPlugin(EmberAfPluginDoorLockWeekDaySchedule &plugin)
```

Comment:
1. Are schedules necessarily related to credentials? Now it looks a little bit like schedules would be a subset of credentials feature. Maybe we should work on some renaming e.g. to make this file describe generally all data types related to door lock features.
2. I think we should add `#ifdef CONFIG_LOCK_SCHEDULES` here to save some flash if schedules are not used.

Reply:
Ehh, we can but it requires changing in many places... We decided to have a credentials manager and now for example the CredentialsManager class is a template class that depends on CredentialsBits. So the whole module needs to be refactored then. I can move schedules to bolt_lock_manager, but then it will not be consistent with credentials_data_types, because they are still `credentials`. So what I can propose is to rename all modules and files from `credentials` to `lock_persistent`. @markaj-nordic  what do you think?

Reply:
2. done.

Reply:
To me, having the scheduled access feature implemented under the credentials module is quite confusing. My understanding is that, we have very similar implementation (because credentials and scheduled access operates in a similar manner and both needs persistence) of high level features that are quite unrelated. The easiest solution I see for now is to rename the CredentialsManager/Data to something more generic, like "AccessManager"/AccessData.

Reply:
f2f, decided to rename it to "Access"